tests/storage/bitmaps_test.py

```diff
@@ -234,6 +236,78 @@ def test_remove_non_existing_bitmap_succeed(tmp_mount, vol_chain):
     bitmaps.remove_bitmap(vol_chain.top_vol, 'bitmap')
 
 
+def test_prune_stale_bitmaps(tmp_mount, vol_chain):
+    # Add valid bitmap to volumes
+    bitmap = 'valid-bitmap'
+    qemuimg.bitmap_add(vol_chain.base_vol, bitmap).run()
+    qemuimg.bitmap_add(vol_chain.top_vol, bitmap).run()
+
+    # Add stale bitmaps to base volume
+    for i in range(5):
+        qemuimg.bitmap_add(vol_chain.top_vol, f"stale-bitmap-{i}").run()
+
+    bitmaps.prune_bitmaps(vol_chain.base_vol, vol_chain.top_vol)
+
+    info = qemuimg.info(vol_chain.base_vol)
+    assert info['format-specific']['data']['bitmaps'] == [
+        {
+            "flags": ['auto'],
+            "name": bitmap,
+            "granularity": 65536
+        },
+    ]
+
+
+def test_prune_disabled_bitmaps(tmp_mount, vol_chain):
+    # Add valid bitmap to volumes
+    bitmap = 'valid-bitmap'
+    qemuimg.bitmap_add(vol_chain.base_vol, bitmap).run()
+    qemuimg.bitmap_add(vol_chain.top_vol, bitmap).run()
+
+    # Add disabled bitmaps to volumes
+    for i in range(5):
+        bitmap_disabled = f"disabled-bitmap-{i}"
+        qemuimg.bitmap_add(vol_chain.base_vol, bitmap_disabled).run()
+        qemuimg.bitmap_add(
+            vol_chain.top_vol, bitmap_disabled, enable=False).run()
+
+    bitmaps.prune_bitmaps(vol_chain.base_vol, vol_chain.top_vol)
+
+    info = qemuimg.info(vol_chain.base_vol)
+    assert info['format-specific']['data']['bitmaps'] == [
+        {
+            "flags": ['auto'],
+            "name": bitmap,
+            "granularity": 65536
+        },
+    ]
+
+
+def test_prune_in_use_bitmaps(tmp_mount, vol_chain):
+    # Add inconsistent "in-use" bitmaps to volumes
+    for i in range(5):
+        bitmap_in_use = f"in-use-bitmap-{i}"
+        qemuimg.bitmap_add(vol_chain.base_vol, bitmap_in_use).run()
+        qemuimg.bitmap_add(vol_chain.top_vol, bitmap_in_use).run()
+        qemuio.abort(vol_chain.top_vol)
```

Comment:
We can run qemuio.abort() once after adding all the bitmaps. It will break all bitmaps in the volume.

Reply:
I knew since I tried before, but still indented this wrongly... Fixing.